controller/src/plan.rs

```diff
@@ -21,7 +21,7 @@ pub struct ControllerPlan {
 
 impl ControllerPlan {
     pub async fn from_controller_config(config: ControllerConfig) -> Result<Self> {
-        let nats = config.nats.connect_with_retry().await?;
+        let nats = config.nats.connect_with_retry("controller.inbox").await?;
```

Comment:
so If I understand correctly, responses will be published to "controller.inbox.<UID>" ?

Reply:
It will be `controller.inbox.{some random string generated by NATS}`